new/LIT6907MastabqMar.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6907MastabqMar" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መስተብቍዕ፡ ዘማርያም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mastabqʷǝʿ za-Māryām</title>
+                <title xml:id="t2">Mastabqʷǝʿ za-Māryām II</title>
+                <title xml:lang="gez" xml:id="t3">ወካዕበ፡ ናስተበቍዓ፡ ለንግሥተ፡ ኵልነ፡</title>
+                <title xml:lang="gez" corresp="#t3">wa-kāʿba nāstabaqqʷǝʿā la-nǝgǝśta kʷǝllǝna</title>
+                <title xml:lang="gez" xml:id="t4">ዓዲ፡ መስተብቍዕ፡ ዘማርያም፡</title>
+                <title xml:lang="gez" corresp="#t4" type="normalized">ʿĀdi Mastabqʷǝʿ za-Māryām</title>
```

Comment:
mastabqʷǝʿ 
same capitalisation issues in the records below

Reply:
Yes, I changed it in all of the three records.